declarations/TikTok.json

```
@@ -2,7 +2,7 @@
   "name": "TikTok",
   "documents": {
     "Terms of Service": {
-      "fetch": "https://www.tiktok.com/legal/terms-of-service?lang=fr",
+      "fetch": "https://www.tiktok.com/legal/terms-of-service-eea?lang=fr",
       "select": [
         ".article-wrapper"
       ]
```

It is necessary to clean the version ;)
```suggestion
"select": [
        ".article-wrapper"
      ],
      "remove": [
        ".language-selection",
        ".dropdown-title > img"
      ]
```